o2a/mappers/dummy_mapper.py

```diff
@@ -14,16 +14,37 @@
 # limitations under the License.
 """Dummy Mapper that is used as temporary solution while we are implementing the real mappers.
 """
-from typing import Set
+from typing import Set, Tuple, List
+from xml.etree.ElementTree import Element
 
+from o2a.converter.relation import Relation
 from o2a.converter.task import Task
 from o2a.mappers.action_mapper import ActionMapper
+from o2a.utils.dict_utils import remove_key_from_dictionary_copy
 
 
 class DummyMapper(ActionMapper):
-    def to_tasks_and_relations(self):
-        tasks = [Task(task_id=self.name, trigger_rule=self.trigger_rule, template_name="dummy.tpl")]
-        relations = []
+    """Dummy mapper used in place of not-yet-implemented mappers """
+
+    def __init__(self, oozie_node: Element, name: str, dag_name: str, **kwargs):
+        kwargs, job_properties = remove_key_from_dictionary_copy(kwargs, "job_properties", {})
+        kwargs, configuration_properties = remove_key_from_dictionary_copy(
+            kwargs, "configuration_properties", {}
+        )
+        super().__init__(
+            oozie_node=oozie_node,
+            name=name,
+            dag_name=dag_name,
+            job_properties=job_properties,
+            configuration_properties=configuration_properties,
+            **kwargs,
+        )
+
+    def to_tasks_and_relations(self) -> Tuple[List[Task], List[Relation]]:
```

Comment:
The parent class contains type definitions. Is it necessary?

Reply:
Again - I prefer to repeat it for readability